contracts/erc20/package.json

```diff
@@ -23,7 +23,7 @@
         "watch": "sol-compiler -w",
         "clean": "shx rm -rf lib test/generated-artifacts test/generated-wrappers generated-artifacts generated-wrappers",
         "generate_contract_wrappers": "abi-gen --debug --abis  ${npm_package_config_abis} --output test/generated-wrappers --backend ethers",
-        "lint": "tslint --format stylish --project . --exclude ./generated-wrappers/**/* --exclude ./test/generated-wrappers/**/* --exclude ./generated-artifacts/**/* --exclude ./test/generated-artifacts/**/* --exclude **/lib/**/* && yarn lint-contracts",
+        "lint": "yarn lint-contracts",
```

Comment:
I think this is fine since we will move away from tslint. 

maybe add a `TODO` to add eslint back (instead of just removing it)
`"lint": "yarn lint-contracts # TODO: add eslint"`